SpaDES4Dummies.Rmd

```diff
@@ -10,116 +10,142 @@ output: html_document
 This guide will take you through how to start your own model using **`SpaDES`**. It assumes you have already installed **`SpaDES`** and that it is working in your machine. If you haven't done this yet, please have a look at [this gist](https://gist.github.com/tati-micheletti/1de7505cdd387fe997f127f13eeb4393) to bypass known issues.
 
 
-
-### The example
+### THE EXAMPLE
 Let's imagine we want to understand whether the abundance of a species is statistically related with temperature. Both the abundance data and the temperature data are being constantly updated. Also, we want to have the possibility of analysing the relationship between the two iteratively, without needing to constantly change our script to account for the new data inputs.
 In this example the abundance and temperature data are being updated by a simulation model.
 
 
-### Before...
+### BEFORE `SpaDES`...
 If we use R to develop our species abundance and temperature simulation models in the 'conventional way', we'll probably have a script were everything happens - the simulations and data analysis. At most, we would have a main script that sources others that contain useful functions, or are doing the simulations and data treatment/analysis separately. If you already use R like this, you'll find that the **`SpaDES`** way of thinking is similar. So why use it? Because it makes changing, adapting and sharing your code - or modules - much easier. 
 
 Still don't believe me? Okay, let's solve our problem using the conventional way:
 
-```{r the_r_way}
+#### SETUP
+Load library raster without messages:
+```{r supress}
 suppressMessages(library(raster))
+```
 
-## Template raster
+Create a raster template:
+```{r the_r_way_raster}
 r <- raster(nrows = 100, ncols = 100, xmn = -50, xmx = 50, ymn = -50, ymx = 50)
+```
 
-## SPECIES ABUNDANCE "SIMULATIONS"
-## Our VERY simple "simulation" model merely generates rasters that follows a Gaussian distribution
+#### SPECIES ABUNDANCE "SIMULATIONS"
+Our VERY simple "simulation" model (in form of a function) merely generates rasters that follows a Gaussian distribution
```

Comment:
instead:
Our VERY simple "simulation" model (in form of a function) generates rasters that follow a Gaussian distribution